docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  myapp:
+    build:
+      context: .
+      dockerfile: Dockerfile.bitbucket_pipeline
+    ports:
+      - "8080:80"
+    environment:
+      - BITBUCKET_BRANCH=${BITBUCKET_BRANCH}
+      - BITBUCKET_BUILD_NUMBER=${BITBUCKET_BUILD_NUMBER}
+      - BITBUCKET_CLONE_DIR=${BITBUCKET_CLONE_DIR}
+      - BITBUCKET_COMMIT=${BITBUCKET_COMMIT}
+      - BITBUCKET_GIT_HTTP_ORIGIN=${BITBUCKET_GIT_HTTP_ORIGIN}
+      - BITBUCKET_GIT_SSH_ORIGIN=${BITBUCKET_GIT_SSH_ORIGIN}
+      - BITBUCKET_PIPELINE_UUID=${BITBUCKET_PIPELINE_UUID}
+      - BITBUCKET_PROJECT_KEY=${BITBUCKET_PROJECT_KEY}
+      - BITBUCKET_PROJECT_UUID=${BITBUCKET_PROJECT_UUID}
+      - BITBUCKET_PR_DESTINATION_BRANCH=${BITBUCKET_PR_DESTINATION_BRANCH}
+      - BITBUCKET_PR_DESTINATION_COMMIT=${BITBUCKET_PR_DESTINATION_COMMIT}
+      - BITBUCKET_PR_ID=${BITBUCKET_PR_ID}
+      - BITBUCKET_REPO_FULL_NAME=${BITBUCKET_REPO_FULL_NAME}
+      - BITBUCKET_REPO_IS_PRIVATE=${BITBUCKET_REPO_IS_PRIVATE}
+      - BITBUCKET_REPO_OWNER=${BITBUCKET_REPO_OWNER}
+      - BITBUCKET_REPO_OWNER_UUID=${BITBUCKET_REPO_OWNER_UUID}
+      - BITBUCKET_REPO_SLUG=${BITBUCKET_REPO_SLUG}
+      - BITBUCKET_REPO_UUID=${BITBUCKET_REPO_UUID}
+      - BITBUCKET_SSH_KEY_FILE=${BITBUCKET_SSH_KEY_FILE}
+      - BITBUCKET_STEP_RUN_NUMBER=${BITBUCKET_STEP_RUN_NUMBER}
+      - BITBUCKET_STEP_TRIGGERER_UUID=${BITBUCKET_STEP_TRIGGERER_UUID}
+      - BITBUCKET_STEP_UUID=${BITBUCKET_STEP_UUID}
+      - BITBUCKET_WORKSPACE=${BITBUCKET_WORKSPACE}
+      - CI=${CI}
+      - DOCKER_HOST=${DOCKER_HOST}
+      - PIPELINES_JWT_TOKEN=${PIPELINES_JWT_TOKEN}
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - BITBUCKET_BEARER_TOKEN=${BITBUCKET_BEARER_TOKEN}
```

Comment:
**Suggestion:** Consider using version '3.8' or later for the Docker Compose file to take advantage of the latest features and options.
```suggestion
version: '3.8'
services:
  myapp:
    build:
      context: .
      dockerfile: Dockerfile.bitbucket_pipeline
    ports:
      - "8080:80"
    environment:
      - BITBUCKET_BRANCH=${BITBUCKET_BRANCH}
      - BITBUCKET_BUILD_NUMBER=${BITBUCKET_BUILD_NUMBER}
      - BITBUCKET_CLONE_DIR=${BITBUCKET_CLONE_DIR}
      - BITBUCKET_COMMIT=${BITBUCKET_COMMIT}
      - BITBUCKET_GIT_HTTP_ORIGIN=${BITBUCKET_GIT_HTTP_ORIGIN}
      - BITBUCKET_GIT_SSH_ORIGIN=${BITBUCKET_GIT_SSH_ORIGIN}
      - BITBUCKET_PIPELINE_UUID=${BITBUCKET_PIPELINE_UUID}
      - BITBUCKET_PROJECT_KEY=${BITBUCKET_PROJECT_KEY}
      - BITBUCKET_PROJECT_UUID=${BITBUCKET_PROJECT_UUID}
      - BITBUCKET_PR_DESTINATION_BRANCH=${BITBUCKET_PR_DESTINATION_BRANCH}
      - BITBUCKET_PR_DESTINATION_COMMIT=${BITBUCKET_PR_DESTINATION_COMMIT}
      - BITBUCKET_PR_ID=${BITBUCKET_PR_ID}
      - BITBUCKET_REPO_FULL_NAME=${BITBUCKET_REPO_FULL_NAME}
      - BITBUCKET_REPO_IS_PRIVATE=${BITBUCKET_REPO_IS_PRIVATE}
      - BITBUCKET_REPO_OWNER=${BITBUCKET_REPO_OWNER}
      - BITBUCKET_REPO_OWNER_UUID=${BITBUCKET_REPO_OWNER_UUID}
      - BITBUCKET_REPO_SLUG=${BITBUCKET_REPO_SLUG}
      - BITBUCKET_REPO_UUID=${BITBUCKET_REPO_UUID}
      - BITBUCKET_SSH_KEY_FILE=${BITBUCKET_SSH_KEY_FILE}
      - BITBUCKET_STEP_RUN_NUMBER=${BITBUCKET_STEP_RUN_NUMBER}
      - BITBUCKET_STEP_TRIGGERER_UUID=${BITBUCKET_STEP_TRIGGERER_UUID}
      - BITBUCKET_STEP_UUID=${BITBUCKET_STEP_UUID}
      - BITBUCKET_WORKSPACE=${BITBUCKET_WORKSPACE}
      - CI=${CI}
      - DOCKER_HOST=${DOCKER_HOST}
      - PIPELINES_JWT_TOKEN=${PIPELINES_JWT_TOKEN}
      - OPENAI_API_KEY=${OPENAI_API_KEY}
      - BITBUCKET_BEARER_TOKEN=${BITBUCKET_BEARER_TOKEN}
```

Reply:
What's the purpose of docker compose here? Maybe you added it to the repo by mistake? It's not documented anywhere and I think it's not necessary. It's the responsibility of the user to host the docker image properly. We might need to add better documentation about it in the future